syntax = "proto3";
package fixbug;

message RpcHeader {
    bytes serviceName = 1;
    bytes methodName = 2;
    uint32 arglen = 3;
}